tools/__tasks__/compile/hash/index.js

```diff
@@ -106,7 +106,7 @@ module.exports = {
                             )
                         );
                     })
-                    .then(normalisedAssetMap => // save the asset map
+                    .then((normalisedAssetMap) => // save the asset map
```

Comment:
I guess just still under dev? https://github.com/prettier/prettier/releases/tag/1.2.0

could raise that as a bug?